pkg/utils/yaml.go

```diff
@@ -97,3 +97,84 @@ func LoadYAML(filePath string) ([]*IacDocument, error) {
 
 	return iacDocumentList, nil
 }
+
+// LoadYAMLString loads a YAML String. Can return one or more IaC Documents.
+// Besides reading in file data, its main purpose is to determine and store line number and filename metadata
+func LoadYAMLString(data, absFilePath string) ([]*IacDocument, error) {
```

Comment:
I see a lot of code duplication between `LoadYaml` and `LoadYamlString` methods.

Would it make sense to have something like  `LoadYamlBytes` and let `LoadYamlString` and `LoadYaml` both call `LoadYamBytes`?

Reply:
Yeah good point